### PR TITLE
Fix Travis tests

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ env:
 addons:
   postgresql: '9.4'
 before_install:
-- rvm @global do gem uninstall bundler -a -x
-- rvm @global do gem install bundler -v 1.15.4
+  - gem install bundler -v 1.15.4
+  - alias bundle=bundle _1.15.4_
```
Comment (gmcculloug) [inline .travis.yml]: @bdunne Does this need to be in quotes? 
`alias bundle="bundle _1.15.4_"`

Example: https://github.com/rspec/rspec-dev/blob/master/travis/.travis.yml#L12

**Comment by simaishi** [comment]
Euwe backport (to manageiq repo) details:

```
$ git log -1
commit 50caa577a7d2348f530a49ccf21d607e36a36e66
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Tue Nov 7 15:28:36 2017 -0500

    Merge pull request #108 from bdunne/travis
    
    Fix Travis tests
    (cherry picked from commit 5d5b419607e4144be8704f75a0529db1b3a493bb)
```

**Comment by simaishi** [comment]
Fine backport (to manageiq repo) details:

```
$ git log -1
commit 71946f8b2fe756309822b66d8987fb27ceb86e8f
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Tue Nov 7 15:28:36 2017 -0500

    Merge pull request #108 from bdunne/travis
    
    Fix Travis tests
    (cherry picked from commit 5d5b419607e4144be8704f75a0529db1b3a493bb)
```

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit a5f7355297304f0eb6910ad098ab11883f977bf9
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Tue Nov 7 15:28:36 2017 -0500

    Merge pull request #108 from bdunne/travis
    
    Fix Travis tests
    (cherry picked from commit 5d5b419607e4144be8704f75a0529db1b3a493bb)
```

**Pull request merged**